src/tests/ftest/pool/create_all_vm.py

```diff
@@ -37,7 +36,7 @@ def get_available_bits(self, ranks=[]):
             host_size = len(host_storage["hosts"].split(','))
             for scm_devices in host_storage["storage"]["scm_namespaces"]:
                 rank = scm_devices["mount"]["rank"]
-                if len(ranks) > 0 and rank not in ranks:
+                if ranks and len(ranks) > 0 and rank not in ranks:
```

Comment:
Thanks for fixing! FYI, that `in` works for an empty list too
```suggestion
                if ranks and rank not in ranks:
```

Reply:
- [ ] Remove useless tests